tests/monitoring/monitoring.go

```diff
@@ -637,8 +642,60 @@ var _ = Describe("[Serial][sig-monitoring]Prometheus Alerts", Serial, decorators
 			libwait.WaitForVirtualMachineToDisappearWithTimeout(vmi, 240)
 		})
 	})
+
+	Context("System Alerts", func() {
+		BeforeEach(func() {
+			scales = make(map[string]*autoscalingv1.Scale, 1)
+			backupScale(virtOperator.deploymentName)
+			updateScale(virtOperator.deploymentName, 0)
+
+			deleteDaemonSet(virtHandler.deploymentName)
+			reduceAlertPendingTime()
+		})
+
+		AfterEach(func() {
+			revertScale(virtOperator.deploymentName)
+			waitUntilAlertDoesNotExist("KubeVirtNoAvailableNodesToRunVMs")
+		})
+
+		It("KubeVirtNoAvailableNodesToRunVMs should be triggered when there are no available nodes in the cluster to run VMs.", func() {
```

Comment:
I would also check that alert disappears when `allowEmulation` is true

Reply:
added